src/Factory/DomainFactory.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Factory;
+
+use App\Entity\Domain;
+use App\Repository\DomainRepository;
+use Zenstruck\Foundry\ModelFactory;
+use Zenstruck\Foundry\Proxy;
+use Zenstruck\Foundry\RepositoryProxy;
+
+/**
+ * @method static       Domain|Proxy createOne(array $attributes = [])
+ * @method static       Domain[]|Proxy[] createMany(int $number, $attributes = [])
+ * @method static       Domain|Proxy find($criteria)
+ * @method static       Domain|Proxy findOrCreate(array $attributes)
+ * @method static       Domain|Proxy first(string $sortedField = 'id')
+ * @method static       Domain|Proxy last(string $sortedField = 'id')
+ * @method static       Domain|Proxy random(array $attributes = [])
+ * @method static       Domain|Proxy randomOrCreate(array $attributes = [])
+ * @method static       Domain[]|Proxy[] all()
+ * @method static       Domain[]|Proxy[] findBy(array $attributes)
+ * @method static       Domain[]|Proxy[] randomSet(int $number, array $attributes = [])
+ * @method static       Domain[]|Proxy[] randomRange(int $min, int $max, array $attributes = [])
+ * @method static       DomainRepository|RepositoryProxy repository()
+ * @method Domain|Proxy create($attributes = [])
+ */
+final class DomainFactory extends ModelFactory
+{
+    protected function getDefaults(): array
+    {
+        return [
+            'url' => self::faker()->company(),
```

Comment:
```suggestion
            'url' => self::faker()->domainName(),
```

Use `domainName` instead
https://fakerphp.github.io/formatters/internet/#domainname

Reply:
Way better thank you 👍